specs/mediawiki/v1/content.yaml

```diff
@@ -788,6 +849,8 @@ paths:
             $ref: '#/definitions/problem'
       x-backend-request:
         uri: /{domain}/sys/parsoid/transform/html/to/wikitext{/title}{/revision}
+        body:
+          html: '{html}'
```

Comment:
Actually, `oldid` is being translated by the VRS, so we need to add just the `scrubWikitext` flag